### ISSUE TITLE
Social media icons in footer not focusable with touch exploration on VoiceOver iOS. You can't touch them you can only swipe to them.

### ISSUE THREAD

**Issue by JohnHewittCriterion** [flexion/fs-intake-module#831]
Footer social media icons are not focusable with touch exploration on iOS voiceover because the accessibility technique is negative positioned visually hidden CSS method which is not supported for mobile touch because the user can't touch -999em off the screen with their fingers.
 
http://pauljadam.com/demos/positionbug.html
 
Suggest SVG for icons and aria-label for link accessible name.
http://pauljadam.com/demos/svg.html
http://pauljadam.com/demos/linkpurpose.html 

![image](https://user-images.githubusercontent.com/28604890/30262386-b4fd494a-969e-11e7-8ab1-4e8a8efc532d.png)

![image](https://user-images.githubusercontent.com/28604890/30262391-baa6a63e-969e-11e7-951e-940009d6ce68.png)

WCAG: 1.3 Adaptable - 1.3.2 Meaningful Sequence 
EPERM-156

**Comment by npage123** [comment]
-[ ] Check with Mac Voice Over
-[ ] Check with JAWS
